cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,16 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => {
+    cy.visit(`https://react-redux.realworld.io/#/login`);
+  });
   it('should provide an ability to log in', () => {
-
+    cy.get(':nth-child(1) > .form-control')
+      .type('jjoohh@gmail.com');
+    cy.get(':nth-child(2) > .form-control')
+      .type('qwert12345');
+    cy.get('.btn').click();
+    cy.get('.navbar').last()
+      .should('contain.text', 'ttetetetfdghfdg');
```

Comment:
```suggestion
      .should('contain.text', 'ttetetetfdghfdg');
```
use for test more readable test data pls